### PR TITLE
Us 1 game init

### PR DESCRIPTION
**Pull request by maxime-auneau**

### CODE REVIEW

**Review by AshBll — DISMISSED** [review]
ok

**Review by FranciscoKevin — DISMISSED** [review]
OK

File: src/View/layout.html.twig [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 {% block content %}{% endblock %}
 {% block javascript %}{% endblock %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (RomainClair) [inline src/View/layout.html.twig]: Do you really need jquery ?!?

**Review by FranciscoKevin — DISMISSED** [review]
C'est bon

**Review by thocle86 — COMMENTED** [review]
Ok

**Review by FranciscoKevin — DISMISSED** [review]
Yes good.

**Review by thocle86 — COMMENTED** [review]
ok

**Review by thocle86 — DISMISSED** [review]
ok

File: src/Controller/GameController.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: aurelwcs
+ * Date: 08/04/19
+ * Time: 18:40
+ */
```
Comment (RomainClair) [inline src/Controller/GameController.php]: Seems not right =)

File: src/Model/TroopManager.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sylvain
+ * Date: 07/03/18
+ * Time: 18:20
+ * PHP version 7
+ */
```
Comment (RomainClair) [inline src/Model/TroopManager.php]: Also not true

File: src/Model/TroopManager.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    public function insert(Troop $troop): int
+    {
+        // prepared request
+            $insert = $this->pdo->prepare("INSERT INTO " . self::TABLE . " (name, strength) VALUES (:name, :strength)");
+            $insert->bindValue('name', $troop->getName(), PDO::PARAM_STR);
+            $insert->bindValue('strength', $troop->getLevel(), PDO::PARAM_INT);
+        if ($insert->execute()) {
+            return (int)$this->pdo->lastInsertId();
+        }
```
Comment (RomainClair) [inline src/Model/TroopManager.php]: You should check for errors becaus PDO::prepare, PDOStatement::bindValue and PDOStatement::execute can return false

File: src/Model/TroopManager.php [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    public function delete(): void
+    {
+        // prepared request
+        $truncate = $this->pdo->prepare("TRUNCATE " . self::TABLE);
+        $truncate->execute();
```
Comment (RomainClair) [inline src/Model/TroopManager.php]: Check for errors

**Review by RomainClair — CHANGES_REQUESTED** [review]
You probably should have a comment block at the beginning of every class and before each method to explain what it is and how to use it

**Review by thocle86 — APPROVED** [review]
Ok

**Review by FranciscoKevin — APPROVED** [review]
C'est bon

**Review by RomainClair — APPROVED** [review]

**Pull request merged**